server/models/Profile.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require("mongoose");
+const userSchema = require("./User");
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  dob: {
+    type: Date,
+    required: true,
+  },
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+  },
+  userPhotoUrl: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  availability: {
+    type: Array,
```

Comment:
This will be `type: [String]` here :)

Reply:
made the change and thanks for the heads up